specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/examples/getDeployments.json

```
@@ -0,0 +1,72 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "workspaceName": "workspaces123",
+    "inferenceEndpointName": "testendpoint",
+    "api-version": "2020-09-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/testrg123/providers/Microsoft.MachineLearningServices/workspaces/workspaces123/inferenceEndpoints/testendpoint/deployments/testdeployment",
+            "name": "testdeployment",
+            "identity": "",
+            "location": "westus2euap",
+            "type": "Microsoft.MachineLearningServices/workspaces/inferenceEndpoints/deployments",
+            "sku": {
+              "name": "Standard_DS3_V2",
+              "tier": "Basic",
+              "capacity": 5
+            },
+            "properties": {
+              "modelIds": "modelURL",
```

This should be an array instead of a string
```suggestion
              "modelIds": ["modelURL"],
```
Please remember run prettier after you make this change